src/redux/store.js

```diff
@@ -0,0 +1,21 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: bookReducer,
+});
+
+export default { store };
```

Comment:
Great job on creating your redux store. However, here are some issues and suggestions for improvement:

- 🚨 You have used `configureStore` to create a reducer, but you have not imported your categories reducer and only passed `bookReducer` as the value of the `reducer` option. This means that your store will only have one slice of state, which is not what the project requires. To fix this, you need to pass an object with both `books` and `categories` as the keys, and their corresponding reducers as the values. This will automatically use `combineReducers` to create a root reducer that combines both slices of state. For example, you should write:

```js
import { configureStore } from "@reduxjs/toolkit"
import books from "./books/booksSlice"
import categories from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books, categories },
})
```

Source: Conversation with Bing, 04/09/2023
(1) configureStore | Redux Toolkit. https://redux-toolkit.js.org/api/configurestore/.
(2) Configuring Your Store | Redux. https://redux.js.org/usage/configuring-your-store.
(3) redux - Redux Toolkit - configureStore() - Web Coding Center. https://webcodingcenter.com/redux/configureStore.
(4) Converting the Store to Use `configureStore()` - Codecademy. https://www.codecademy.com/courses/learn-redux/lessons/the-redux-toolkit/exercises/converting-the-store-to-use-configurestore.
(5) Usage Guide | Redux Toolkit - JS.ORG. https://redux-toolkit.js.org/usage/usage-guide/.

Reply:
Thanks a lot.